runtime/src/main/java/uk/ac/manchester/tornado/runtime/common/TornadoOptions.java

```diff
@@ -158,6 +158,12 @@ public static boolean isProfilerEnabled() {
      */
     public static final int TORNADO_SKETCHER_THREADS = Integer.parseInt(getProperty("tornado.sketcher.threads", "4"));
 
+    /**
+     * It enables automatic discovery and parallelisation of loops.
+     * Please note that this option experimental and may cause issues if enabled.
```

Comment:
Add "is" -> Please note that this option is experimental and may cause issues if enabled.